doc/crypto/EVP_EncodeInit.pod

```diff
@@ -76,6 +77,9 @@ in B<*outl>. It is the caller's responsibility to ensure that B<out> is
 sufficiently large to accommodate the output data which will never be more than
 65 bytes plus an additional NUL terminator (i.e. 66 bytes in total).
 
+EVP_ENCODE_CTX_copy() can be used to copy a context B<sctx> to a context
+B<dctx> which must be initialized before calling this function.
```

Comment:
which does which refer to?  make two sentence and say "xxx must be initialized before..." where xxx is sctx or dctx.